### PR TITLE
LDK-revamp new lens initialization file

### PR DESCRIPTION
**Pull request by nazligungor**
Lens initialization script file that creates a new lens project:
- new directory with template files
- package.json file with dependencies and scripts of LDK
- unit tests for util script

### CODE REVIEW

File: bin/lenscompile.js [hunk 1]
```diff
@@ -0,0 +1,108 @@
+#! /usr/bin/env node
+
+/**
+ * bin/lenscompile.js
```
Comment (iamigo) [inline bin/lenscompile.js]: camelCase preferred for file names, so this file should be named "lensCompile.js"

File: bin/lensinit.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
+#! /usr/bin/env node
+
+/**
+ * bin/lens-init.js
```
Comment (iamigo) [inline bin/lensinit.js]: camelCase the file name, and the name here in the comment should match the file name

File: bin/lenszip.js [hunk 3]
```diff
@@ -0,0 +1,57 @@
+#! /usr/bin/env node
+
+/**
+ * bin/lenszip.js
```
Comment (iamigo) [inline bin/lenszip.js]: camelCase the file name

**Comment by pallavi2209** [comment]
Please include copyright header in each new file

```
/**
 * Copyright (c) 2019, salesforce.com, inc.
 * All rights reserved.
 * Licensed under the BSD 3-Clause license.
 * For full license text, see LICENSE.txt file in the repo root or
 * https://opensource.org/licenses/BSD-3-Clause
 */
```

**Comment by pallavi2209** [comment]
@nazligungor It looks like you have all the changes in #15 . Do you want to keep this PR open and get reviewed as well?
If the other PR includes all the changes, I would suggest you close this PR after making sure that you have resolved the comments in other PR.

**Comment by nazligungor** [comment]
@pallavi2209 I think other PR includes all the changes. Sorry for the confusion. I'll close this one and I just fixed the bug related to compile so everything in the other PR is resolved now.

**Pull request closed without merging**